google_shopping_functions.js

```diff
@@ -5,26 +8,78 @@ var getItemsCount = function(itemData){
 	return itemData.items.length;
 };
 
-//
-//DEFINE FUNCTIONS HERE
-//
+// Question 1
+var getItems = function(objectData) {
+	return objectData.items;
+}
 
 
+console.log(getItems(data));
 
+//Question 2
 
+var items = getItems(data); 
+var getItemsbyBrand = function(items, brand) {
+	var brandItems = [];
+	for (var i = 0; i < data["items"].length; i++ ) {
+	if (data["items"][i].product.brand === brand) {
```

Comment:
This will work, but it's not making use of your parameter (items).
Your parameter is a placeholder for whatever argument it takes, so you would want to use it as if it were a template, in this fashion:

```
if (items[i].product.brand === brand)
```